src/main/java/meteordevelopment/meteorclient/systems/modules/player/AutoTool.java

```diff
@@ -123,6 +129,7 @@ private void onStartBreakingBlock(StartBreakingBlockEvent event) {
 
         for (int i = 0; i < 9; i++) {
             double score = getScore(mc.player.getInventory().getStack(i), blockState, silkTouchForEnderChest.get(), prefer.get(), itemStack -> !shouldStopUsing(itemStack));
+            if (blacklist.get().contains(mc.player.getInventory().getStack(i).getItem())) score = -1;
```

Comment:
Can you move this before the `getScore()` call and return early so it doesn't need to run if it is already blacklisted. And while you are at it put the item inside a variable just so it is easier to read than twice repeating `mc.player.getInventory().getStack(i)`.